### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by JakeDaBoss**
- Fixes Medical Examiner position.
- Fixes to other wiring issues on upper med and command.

### CODE REVIEW

File: html/changelog.html [hunk 1]
```diff
@@ -77,6 +77,8 @@ <h3 class="author">OliverCarswell updated:</h3>
 				<li class="bugfix">Wiring issues fixed (hopefully).</li>
 				<li class="bugfix">Fixed improper wiring in Morgue having the wrong knots.</li>
 				<li class="bugfix">Fixed three area's not having air alarms.</li>
+				<li class="bugfix">Fixed Medical Examiner.</li>
+				<li class="bugfix">Fixed upper Command power issues.</li>
```
Comment (BlueNexus) [inline html/changelog.html:81]: Please do not edit the main changelog directly.

Comment (JakeDaBoss) [reply html/changelog.html:81]: I've not. I made a new .yml and ran the tool. It just so adds it to the date.

**Review by BlueNexus — APPROVED** [review]
I'll approve this, but don't make a habit of directly editing the main changelog.

File: html/changelog.html (hunk 1 above)
Comment (BlueNexus) [reply html/changelog.html:81]: 👍

**Comment by JakeDaBoss** [comment]
Like I said, I haven't. I've made a new .yml.

**Pull request merged**